fedimint-client/Cargo.toml

```diff
@@ -42,6 +42,7 @@ thiserror = { workspace = true }
 tokio = { version = "1.36.0", features = [ "time", "macros" ] }
 tokio-stream = { version = "0.1.14", features = [ "time", "sync" ] }
 tracing = { workspace = true }
+reqwest = { version = "0.12.2", features = ["json", "rustls-tls"], default-features = false }
```

Comment:
:/ I wonder how heavy it is.

Reply:
we already use reqwest on client side, through jsonrpsee

Reply:
It's also already used by the LN module to do the same